src/App.tsx

```diff
@@ -1,109 +1,37 @@
 import React from 'react';
-
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+
+import { PostList } from './components/PostList';
+import { Post } from './types/Post';
+import { Comment } from './types/Comment';
+import { User } from './types/User';
+import { FullPost } from './types/FullPost';
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+const preparePosts = (
+  posts: Post[],
+  comments: Comment[],
+  users: User[],
+): FullPost[] => {
+  return posts.map(post => ({
```

Comment:
you can return something immediately, by using usual brackets instead of curly and without `return` at all